### PR TITLE
LPS-69733 Receiving the condition's operators from the back end

### PR DESCRIPTION
**Pull request by thiago-rocha**

### CODE REVIEW

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_render_rule_condition.js [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+				var instance = this;
+
+				var value;
+
+				var operators = [];
+
+				if (condition) {
+					value = condition.operator;
+				}
+console.log(instance.get('functionsMetadata'));
```
Comment (diogo-lins) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_render_rule_condition.js]: Remove the console.log

**Comment by diogo-lins** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/config.js [hunk 2]
```diff
@@ -153,7 +153,14 @@
 						},
 						'liferay-ddl-form-builder-rule-builder': {
 							path: 'form_builder_rule_builder.js',
-							requires: ['liferay-ddl-form-builder-render-rule', 'liferay-ddl-form-builder-rule-builder-template']
+							requires: [
+							'liferay-ddl-form-builder-render-rule',
+							'liferay-ddl-form-builder-render-rule-condition',
+							'liferay-ddl-form-builder-rule-builder-template']
+						},
+						'liferay-ddl-form-builder-render-rule-condition': {
```
Comment (diogo-lins) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/config.js]: Sort properties: liferay-ddl-form-builder-rule-builder, liferay-ddl-form-builder-render-rule-condition

**Comment by thiago-rocha** [comment]
Updated.

**Comment by diogo-lins** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by diogo-lins** [comment]
Pull request submitted to https://github.com/ctampoya/liferay-portal/pull/69. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**